src/components/SwitchButton/index.tsx

```diff
@@ -72,6 +109,12 @@ const SwitchButton = ({
   rightButton,
   tooltip,
 }: SwitchButtonProps) => {
+  const leftButtonRef = useRef<HTMLLabelElement | null>(null)
+  const rightButtonRef = useRef<HTMLLabelElement | null>(null)
```

Comment:
Not sure but I think we can remove the `| null`
```suggestion
  const leftButtonRef = useRef<HTMLLabelElement>(null)
  const rightButtonRef = useRef<HTMLLabelElement>(null)
```